src/app/components/Checkbox/index.tsx

```diff
@@ -0,0 +1,72 @@
+import classNames from 'classnames';
+import React, { useCallback, useMemo } from 'react';
+import styles from './index.module.scss';
+
+export enum CheckboxAlignment {
+  LEFT = 'left',
+  RIGHT = 'right',
+}
+
+type CheckboxProps = {
+  label: string | React.ReactNode;
```

Comment:
```suggestion
  label: React.ReactNode;
```
string is already included in ReactNode